common/views/components/Buttons/ButtonControl/README.md

```diff
@@ -0,0 +1,6 @@
+### Button Controls
+Used in application with a UI component, e.g. launch image viewer.
```

Comment:
I can't say I can make sense of this sentence.
Everything is a UI component.

Is there something more specific about controlling elements and components on the page rather than taking you to another place 🚀 ?